src/components/Nav/Nav.tsx

```diff
@@ -75,6 +84,15 @@ export function NavbarScreen() {
               </Fragment>
             )}
           </div>
+          {showProfileMenu && (
```

Comment:
There is no way to open the modal if the user is on a phone

Reply:
![image](https://github.com/hawks-atlanta/frontend-react/assets/131629166/cc419792-d644-41ca-8821-257d76f7c1c8)